vllm/model_executor/models/deepseek_v2.py

```diff
@@ -619,7 +619,15 @@ def sparse_attn_indexer(
     has_prefill = attn_metadata.num_prefills > 0
     num_decode_tokens = attn_metadata.num_decode_tokens
 
-    ops.indexer_k_quant_and_cache(
+    indexer_k_quant_cache_and_cache_func = ops.indexer_k_quant_and_cache
```

Comment:
can you use CustomOp to select the kernel implementation for different platforms?